packages/ckeditor5-modern/src/ModernEditor/ModernEditor.tsx

```diff
@@ -18,16 +18,20 @@ import { ModernEditorProps } from '../types/modern-editor';
 import { ImageIcon } from '../icons/ImageIcon';
 import { MODERN_EDITOR_STYLE } from '../utils/styleUtils';
 import { PostFormSchemaValues, postFormSchema } from './validation';
+import _ from 'lodash';
```

Comment:
@nberongoy treeshaking is supported in tsdx but just export which functions you are going to use with lodash.